StreamChatClient_v3/Controllers/ChannelListController.swift

```diff
@@ -134,6 +134,52 @@ public class ChannelListControllerGeneric<ExtraData: ExtraDataTypes>: Controller
     }
 }
 
+// MARK: - Actions
+
+public extension ChannelListControllerGeneric {
+    /// Delete the channel.
+    /// - Parameters:
+    ///   - channel: The channel you want to delete.
+    ///   - completion: An empty completion block.
+    func delete(channel: ChannelModel<ExtraData>, completion: ((Error?) -> Void)? = nil) {
+        client.apiClient.request(endpoint: .delete(channel: channel)) { result in
+            switch result {
+            case .success: completion?(nil)
+            case .failure(let error): completion?(error)
+            }
+        }
+    }
+
+    /// Hide the channel from queryChannels for the user until a message is added.
+    /// - Parameters:
+    ///   - channel: The channel you want to hide.
+    ///   - userId: Current user Id.
+    ///   - clearHistory: Flag to remove channel history.
+    ///   - completion: An empty completion block.
+    func hide(channel: ChannelModel<ExtraData>, userId: UserId, clearHistory: Bool, completion: ((Error?) -> Void)? = nil) {
+        client.apiClient.request(endpoint: .hide(channel: channel, userId: userId, clearHistory: clearHistory)) { result in
+            switch result {
+            case .success: completion?(nil)
+            case .failure(let error): completion?(error)
+            }
+        }
+    }
+
+    /// Removes hidden status for the specific channel.
+    /// - Parameters:
+    ///   - channel: The channel you want to show.
+    ///   - userId: Current user Id.
+    ///   - completion: An empty completion block.
+    func show(channel: ChannelModel<ExtraData>, userId: UserId, completion: ((Error?) -> Void)? = nil) {
+        client.apiClient.request(endpoint: .show(channel: channel, userId: userId)) { result in
+            switch result {
+            case .success: completion?(nil)
+            case .failure(let error): completion?(error)
+            }
+        }
+    }
+}
```

Comment:
All these 3 functions need to be tested. See https://github.com/GetStream/stream-chat-swift/pull/378#discussion_r465803489